Source/ACE/Entity/Creature.cs

```diff
@@ -164,6 +171,10 @@ public void Kill(Session session)
             corpse.Location.PositionY -= corpse.PhysicsData.ObjScale;
             corpse.Location.PositionZ -= corpse.PhysicsData.ObjScale / 2;
 
+            // Corpses stay on the ground for 5 * player level but minimum 1 hour
+            // corpse.DespawnTime = Math.Max((int)session.Player.PropertiesInt[Enum.Properties.PropertyInt.Level] * 5, 360) + WorldManager.PortalYearTicks; // as in live
+            corpse.DespawnTime = 20 + WorldManager.PortalYearTicks; // only for testing
+
             // Remove Creature from Landblock and add Corpse in that location via the ActionQueue to honor the motion delays
             QueuedGameAction removeCreature = new QueuedGameAction(this.Guid.Full, this, true, true, GameActionType.ObjectDelete);
             QueuedGameAction addCorpse = new QueuedGameAction(this.Guid.Full, corpse, true, GameActionType.ObjectCreate);
```

Comment:
Doesn't the corpse creation need to get broadcast to all players?  And added to the Landblock?